src/Lykke.Payments.Link4Pay/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1
+FROM microsoft/dotnet:2.2-aspnetcore-runtime
```

Comment:
downgrade?

Reply:
it's not easy to use .net core 3.1 with current infrastructure packages